### PR TITLE
Add nollup

### PR DESCRIPTION
**Pull request by guybedford**
Awesome Contribution Checklist:

- [x] I have read, and re-read the [Contributing Guidelines](.github/COONTRIBUTING.md)
- [x] I have searched to ensure the suggested item doesn't exist on this list
- [x] This PR contains only one item

### Please Describe Your Addition

This adds Nollup which otherwise users might not be able to find. Open to including it in another section as appropriate. I haven't personally tested this so can't necessarily vouch for stability but I really like the approach in principle.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
 ### All-Purpose Awesome
 
 - [graph](https://github.com/ondras/rollup-plugin-graph) – Generates a module dependency graph.
+- [nollup](https://github.com/PepsRyuu/nollup) - Rollup development bundler following the same plugin API for hot module reloading.
```
Comment (shellscape) [inline README.md]:
```suggestion
- [nollup](https://github.com/PepsRyuu/nollup) - Rollup-compatible development bundler providing Hot Module Replacement.
```

Comment (guybedford) [reply README.md]: Looks much better, thanks!

Comment (guybedford) [reply README.md]: Woah this GitHub feature is awesome.

Comment (shellscape) [reply README.md]: 👍

**Pull request merged**